src/styles/page-template/_template.njk

```diff
@@ -136,13 +136,15 @@
                                 }}
                             {% endif %}
                             {% block preMain %}{% endblock %}
+                            {% if pageConfig.rtl is defined and pageConfig.rtl %}<div class="rtl">{% endif %}
```

Comment:
Census website templates replace the higher level `pageContent` block. This is due to a legacy issue where the grid was set at `grid-col-8`. This has now been reworked in the DS so that main grid col can be set using `pageColNumber` but to bring this into the generator would require reworking the templates so we replace the `main` block instead.

If this isn't something we want to do now you could add the class to the `body` tag.